lib/appConfig.ts

```diff
@@ -198,14 +198,14 @@ export const createAppConfig = (entries: { [entryAlias: string]: string }, optio
 							return relative('../css', `../${filename}`)
 						}
 						return {
-							// already contains the "js/" prefix as it is our output file configuration
-							runtime: `window.OC.filePath('${options.appName}', '', '${filename}')`,
+							runtime: `window.OC.filePath('${options.appName}', 'js', '${filename}')`,
 						}
 					},
 				},
 				build: {
 					// Output dir is the project root to allow main style to be generated within `/css`
-					outDir: '',
+					outDir: 'js',
+					assetsDir: '../',
```

Comment:
Does not work assets are still put into `js`.
E.g. when using this on the first run wizard it creates:
- `js/css/firstrunwizard-style.css`

Reply:
Weird, it is working with activity. I'll try to reproduce